PhpUnit/AbstractExtensionTestCase.php

```diff
@@ -35,19 +41,45 @@ protected function setUp()
     {
         parent::setUp();
 
+        $this->loadMethodInvoked = false;
+
         foreach ($this->getContainerExtensions() as $extension) {
             $this->container->registerExtension($extension);
         }
     }
 
     /**
+     * Call this method (optionally) form within your test after you have (optionally) modified the ContainerBuilder
+     * for this test ($this->container), but BEFORE you call load method ($this->load()) - if your extension(s) implements
+     * \Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface
+     *
+     * @see http://symfony.com/doc/current/bundles/prepend_extension.html
+     * @see \Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface
+     */
+    protected function prepend()
+    {
+        if ($this->loadMethodInvoked) {
+            throw new \LogicException(sprintf('Method "prepend()" from "%s" must not be invoked after "load()" method.', PrependExtensionInterface::class));
+        }
+
+        foreach ($this->container->getExtensions() as $extension) {
+
+            if ($extension instanceof PrependExtensionInterface) {
+                $extension->prepend($this->container);
+            }
+        }
+    }
+
+    /**
      * Call this method from within your test after you have (optionally) modified the ContainerBuilder for this test
      * ($this->container).
      *
      * @param array $specificConfiguration
      */
     protected function load(array $configurationValues = array())
```

Comment:
That could maybe make developer experience kinda better, and it would maintain BC.

What do you think?

Either way, it should be possible to test load with and without effect of `prepend` invocation.